### ISSUE TITLE
Ghostwriter officially supporting yamado?

### ISSUE THREAD

**Issue by orbifx** [wereturtle/ghostwriter#84]
I'm looking for a good GUI based Markdown editor to integrate with [Yamado](https://github.com/orbifx/yamado).

The idea is to improve the work flow for users who aren't technical enough to configure the necessary scripts. Currently one can set `EDITOR=ghostwriter`, I tried that, but Ghostwriter doesn't support YAML header syntax highlighting. Also it would be nice if Ghostwriter, as a plugin or something, had a sidebar, or options in File or Settings, to hook into Yamado for seamless integration.

In summary:
- [ ] Support YAML headers (benefits working with other processors too like Pandoc)
- [ ] Menu integration for Yamado settings and operations.

Thanks.

**Comment by wereturtle** [comment]
Well, ghostwriter is meant more for editing that one big monster document, like a school paper, dissertation, or a novel.  It's not nearly as much geared toward note-taking and blogs (though it can be used for such, of course).  It's about getting focused on that document you need to write until it finally gets done.

However, I think it would be a great idea to cannibalize ghostwriter for such a note-taking/blogging application.  Yamado looks like it would make a great back-end for such an endeavor.

Unfortunately, I probably won't be doing a such a spin-off of ghostwriter anytime soon.  My time is very limited, and it's all I can do to get 1.4 out the door.  But I think it would be a great idea.  I've been looking for a good app for consolidating notes, preferably one that uses Markdown, but a good and reliable one is hard to find, especially on Linux.  Hopefully, someone out there will be inspired and create what we're looking for.
